python/pyspark/ml/connect/__init__.py

```diff
@@ -31,13 +31,14 @@
     evaluation,
     tuning,
 )
+from pyspark.ml.connect.evaluation import Evaluator
 
 from pyspark.ml.connect.pipeline import Pipeline, PipelineModel
 
 __all__ = [
     "Estimator",
     "Transformer",
-    "Estimator",
+    "Evaluator",
```

Comment:
Oh, is this a bug fix?

Reply:
Yes .. ish .. previously `from pyspark.sql.commect import Evaluator` did not work. Now it works ..

Reply:
Shall we revisit the PR title because this touch the main `ml/connect` code to change the exported symbols?

Reply:
Sure.